### PR TITLE
(editorial) tweak DID relation language

### LINKED ISSUE #60: Odd verbage hard to parse
"DIDs are URLs that relate a DID subject to means for trustable interactions with that subject"

I'm not sure about this sentence as it would be hard to parse for non-native English speakers

### PR DESCRIPTION
**Pull request by TallTed**
fixes #60

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -371,14 +371,14 @@ <h3>
...
-DID subject. This document specifies a common data model, format, and operations
-that all DIDs support.
+authority. DIDs are URLs that relate a DID subject to one or more mechanisms 
+that may be used for trustable interactions with that subject. DIDs resolve 
+to DID Documents, which are simple documents that describe how to use that 
+specific DID. Each DID Document may express cryptographic material, 
+verification methods, and/or service endpoints. These provide a set of 
+mechanisms which enable a DID controller to prove control of the DID. Service 
+endpoints enable trusted interactions with the DID subject. This document 
+specifies a common data model, format, and operations that all DIDs support.
```
Comment (TallTed) [inline index.html]: I'm struggling to find the rewording of this sentence which makes it flow cleanly, and have plural/singular agreement throughout -- `This document specifies a common data model, format, and operations that all DIDs support.`

Perhaps `This document specifies a common data model, a URL format, and a set of operations that all DIDs support.`

**Review by brentzundel — APPROVED** [review]
